target/scripts/build/packages.sh

```diff
@@ -99,16 +99,24 @@ function _install_dovecot() {
     dovecot-core dovecot-imapd
     dovecot-ldap dovecot-lmtpd dovecot-managesieved
     dovecot-pop3d dovecot-sieve dovecot-solr
+    dovecot-auth-lua
   )
 
   if [[ ${DOVECOT_COMMUNITY_REPO} -eq 1 ]]; then
-    _log 'trace' 'Using Dovecot community repository'
-    curl https://repo.dovecot.org/DOVECOT-REPO-GPG | gpg --import
-    gpg --export ED409DA1 > /etc/apt/trusted.gpg.d/dovecot.gpg
-    echo "deb https://repo.dovecot.org/ce-2.3-latest/debian/bullseye bullseye main" > /etc/apt/sources.list.d/dovecot.list
-
-    _log 'trace' 'Updating Dovecot package signatures'
-    apt-get "${QUIET}" update
+    # Dovecot's deb community repository only provides x86_64 packages, so do not include it
+    # when building for another architecture.
+    if [[ "$(uname --machine)" == "x86_64" ]]; then
```

Comment:
I don't think the added level of nesting is necessary here, you should be able to add it as a 2nd condition since there is no other conditional branches to handle.

```bash
# Dovecot's deb community repository only provides x86_64 packages,
# do not include it when building for another architecture
if [[ ${DOVECOT_COMMUNITY_REPO} -eq 1 ]] && [[ "$(uname --machine)" == "x86_64" ]]; then
```

Reply:
A keen eye, and correct. Done. Change will be visible in next commit.